.gitignore

```diff
@@ -1,4 +1,4 @@
-.vscode
```

Comment:
Still ignore `.vscode`, but exclude `.vscode/settings.json`, because there may be other files in .vscode like `tasks.json`.